.github/workflows/certora-prover.yml

```
@@ -0,0 +1,47 @@
+name: Certora Prover Workflow
+
+on:
+  push:
+    branches:
+      - certora
+      - master-rules
+  pull_request:
+    branches:
+      - master
+      - master-report
+      - certora-report
+  workflow_dispatch:
+
+jobs:
+  certora_run:
+    runs-on: ubuntu-latest
+    permissions:
+      contents: read
+      statuses: write
+      pull-requests: write
+      id-token: write
+    steps:
+      - name: Checkout repository
+        uses: actions/checkout@v4
+        with:
+          submodules: recursive
+      - name: Install Foundry
+        uses: foundry-rs/foundry-toolchain@v1
+        with:
+          version: nightly
+      - name: Install forge dependencies
+        run: forge install
+      - name: Munge code
+        run: ./certora/scripts/munge.sh
```

This is our internal terminology, maybe change to something more neutral like just
```suggestion
      - name: Patch code
        run: ./certora/scripts/patch.sh
```